thermal_processing.py

```diff
@@ -167,6 +163,10 @@ def update_metadata(recording, api):
 
 
 def main():
+    processing.init_logging()
+    global conf
+    conf = processing.Config.load()
```

Comment:
This is a bit icky but I guess it'll do for now. A good illustration of why globals are bad :)